### DISCUSSION TITLE
GitHub Projects on mobile app?

### DISCUSSION THREAD

**Discussion by alexbarraboldu** [community/community#8051] [Mobile] [answered]
I think it would be great to have the ability to manage projects on the mobile app.

**Comment by mxie** [comment, marked as answer]
Thanks for your feedback @alexbarraboldu. We're in the process of looking into Projects support already. Check out this [existing discussion](https://github.com/github/feedback/discussions/3023) to provide additional input as well as track future updates from us on this feature.

**Comment by candyho** [comment]
Hey👋just want to inform you we've just shipped [view and edit projects(beta) custom fields]( https://github.blog/changelog/2022-05-09-view-and-edit-projectsbeta-custom-fields-on-github-mobile/) to both iOS and Android last week! You can now edit custom text, number, date, and single-select fields for issues and pull requests that belong to the new projects experience from within the triage sheet. Try it out and let us know what you think! Stay tuned for more updates!
